### PR TITLE
Update organizations.md

### PR DESCRIPTION
**Pull request by stephanieatte**
This PR adds a query to determine the date a user joined an organization

``` graphql query getUserCreated {
  organization(slug: "organization-slug") {
    id
    members(first: 10, search:  "organization-member-name") {
      edges {
        node {
          id
          createdAt
          user {
            id
            name
            email
          }  
        }
      }
    }
  }
}
```

### CODE REVIEW

File: pages/apis/graphql/cookbooks/organizations.md [hunk 1]
```diff
@@ -239,7 +239,30 @@ mutation CreateUser {
 }
 ```
 
+## Get an Organization Member Creation Timestamp
```
Comment (gilesgas) [inline pages/apis/graphql/cookbooks/organizations.md:242]: Thanks for this contribution @stephanieatte ! 🎉 

As per our [writing styleguide](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#headings), please convert this heading to sentence case to allow the docs builds to pass the linting process.

Comment (gilesgas) [inline pages/apis/graphql/cookbooks/organizations.md:242]:
```suggestion
## Get an organization member creation timestamp
```

**Comment by gilesgas** [comment]
Sorry for taking over your PR/branch @stephanieatte to add other docs tweaks I'd been working on recently. I was just hoping to get all of these updates merged in ASAP.

**Review by gilesgas — APPROVED** [review]

**Comment by stephanieatte** [comment]
No worries @gilesgas i was off, just resumed back

**Review by 1atAlcone — APPROVED** [review]

**Review by 1atAlcone — APPROVED** [review]

**Pull request merged**